media/js/firefox/new/scene1-batm-anim.js

```diff
@@ -6,6 +6,7 @@
     'use strict';
 
     var exp = $('#main-header-copy').data('experience');
+    var variant = $('#main-header-copy').data('variant');
```

Comment:
Nit: you can store `$('#main-header-copy')` as a variable to avoid querying the DOM twice here.